contactaccessrights/js/src/access-rights.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  var extPath = CRM.vars.contactAccessRights.baseURL + '/js/src/access-rights';
+  require.config({
+    urlArgs: 'bust=' + (new Date()).getTime(),
+    paths: {
+      'access-rights': extPath
+    }
+  });
+
+  require(['access-rights/app'], function (angular) {
+    var actions = angular.element('.crm-actions-ribbon ul#actions');
+    var app = angular.element('<li id="access-rights"><div id="bootstrap-theme"> \
+        <div ng-controller="AccessRightsCtrl as $ctrl"> \
```

Comment:
Couldn't you put the bootstrap namespace on this element and ditch the wrapper, to keep the markup at a minimum?